tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
+
+Once you have started the Marble stack with the `./pavics-compose.sh up --detach` command, check that everything is
+working properly by opening your webpage in a browser:
+
+- Navigate to the domain name [that you set up](fqdn)
+- Ensure that the page loads as expected
+  - Note that the page that loads should be the one set by the `PROXY_ROOT_LOCATION` variable in `env.local`
+- Ensure that the connection is secure (uses the https protocol with a valid certificate)
+  - If not: ensure that your SSL certificate is up-to-date and that `SSL_CERTIFICATE` variable in `env.local` is
+    pointing to the correct file (see the birdhouse documentation for details)
+
+## Next Steps
+
+Now that you know the basics of how to configure and deploy the Marble stack you can select which optional components
+you would like to add to your stack:
+
+- [Monitoring and Logging components](../monitoring/monitoring.md)
+- [User-facing Services](../services/services.md)
+
+Once you've done that, you will likely want to add your new node to the Marble network:
+
+- [Network](../network/network.md)
+
+If you want to host data on your node:
+
+- [Data hosting](../data/data.md)
+
+And finally you'll want to start adding users to your node:
+
+- [User management](../users/users.md)
+
+## Example installation steps
+
+Here is an example of how to install Marble on a server running Ubuntu 20.04, using mymarble.com as a FQDN, and where we
+get a SSL certificate using [certbot](https://certbot.eff.org/). We assume that [`docker`](https://www.docker.com/) and
+[`git`](https://git-scm.com/) are already installed:
+
+```shell
+# Download source code
+git clone https://github.com/bird-house/birdhouse-deploy.git
+cd birdhouse-deploy/birdhouse
+
+# Configure env.local
+echo "export PAVICS_FQDN=mymarble.com" > env.local
+echo "export DOC_URL='https://marbleclimate.com'" >> env.local
+echo "export MAGPIE_SECRET=$(openssl rand -hex 32)" >> env.local
+echo "export MAGPIE_ADMIN_USERNAME=admin" >> env.local
+echo "export MAGPIE_ADMIN_PASSWORD=$(openssl rand -base64 12)" >> env.local
+echo "export SUPPORT_EMAIL=info@mymarble.com" >> env.local
+echo "export POSTGRES_PAVICS_USERNAME=postgres-pavics" >> env.local
+echo "export POSTGRES_PAVICS_PASSWORD=$(openssl rand -base64 12)" >> env.local
+echo "export POSTGRES_MAGPIE_USERNAME=postgres-magpie" >> env.local
+echo "export POSTGRES_MAGPIE_PASSWORD=$(openssl rand -base64 12)" >> env.local
+
+echo "export EXTRA_CONF_DIRS='...'" >> env.local # fill in the ... with the additional components you want to add
+
+# Get an SSL certificate with certbot
+sudo snap install --classic certbot
+sudo ln -s /snap/bin/certbot /usr/bin/certbot
+sudo certbot certonly --apache --domain mymarble.com # and follow any additional instructions
+
+# Create a file that contains a combination of the fullchain.pem and the privkey.pem files
+sudo cat /etc/letsencrypt/live/mymarble.com/fullchain.pem /etc/letsencrypt/live/mymarble.com/privkey.pem > cert.pem
```

Comment:
How is this /etc/letsencrypt directory created? I wanted to check that we are not missing a step somewhere.

Reply:
It's created by certbot if it doesn't exist already